docs/content/contributing/bicep/bicep-contribution-flow/child-module-publishing.md

```diff
@@ -0,0 +1,173 @@
+* Implement required changes in your fork:
+  * **Allowed list**: If not present, add child module to [child-module-publish-allowed-list.json](https://github.com/Azure/bicep-registry-modules/blob/main/utilities/pipelines/staticValidation/compliance/helper/child-module-publish-allowed-list.json).
+  * **Child module template**: Add `enableTelemetry` parameter and `avmTelemetry` deployment to child main.bicep template.
+  * **Parent module template**: In the main.bicep template of the child module direct parent, add `enableReferencedModulesTelemetry` variable and pass it as `enableTelemetry` value down to the child module deployment.
+  * **Version**: Add the version.json to the child module folder, set version to `0.1`.
+  * **Changelog**: Add a new CHANGELOG.md to the child module folder and update the changelog of all its versioned parents with a new patch version, up to the top-level parent.
+  * **Set-AVMModule**: Run [Set-AVMModule](https://github.com/Azure/bicep-registry-modules/blob/main/utilities/tools/Set-AVMModule.ps1), test your changes, raise a PR.
+
+For more detailed guidance, please refer below.
+
+## Prerequisites
+
+Before jumping into the implementation, make sure the following prerequisites are in place:
+
+### Bicep Child Module Proposal issue
+
+Ensure there is an [AVM issue open](https://github.com/Azure/Azure-Verified-Modules/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22Class%3A%20Child%20Module%20%3Apackage%3A%22%20label%3A%22Language%3A%20Bicep%20%3Amuscle%3A%22) already, proposing the child module to be published. If not, please create one using the [Bicep Child Module Proposal issue template](https://github.com/Azure/Azure-Verified-Modules/issues/new?template=4_module_proposal_bicep_child.yml).
+
+{{% notice style="note" %}}
+
+Please understand the difference between publishing an existing child module and extending a parent module with a not yet implemented child module functionality.
+
+The Bicep Child Module Proposal issue intends to cover the former, so the intended child module MUST already exist in the [BRM (Bicep Registry Modules)](https://aka.ms/BRM) repository source code.
+
+{{% /notice %}}
+
+### Telemetry ID prefix assigned
+
+Follow the below steps to check the child module telemetry ID prefix.
+
+{{% notice style="note" %}}
+
+If the Bicep Child Module Proposal issue was just created, please allow a few days for the telemetry ID prefix to be assigned before reaching out.
+
+{{% /notice %}}
+
+1. Download the Bicep resource module index source CSV file from [here]({{% siteparam base %}}/module-indexes/BicepResourceModules.csv).
```

Comment:
Why download? Would it not be benefitial/easier to use the web view instead? Ref: https://github.com/Azure/Azure-Verified-Modules/blob/main/docs/static/module-indexes/BicepResourceModules.csv

Reply:
good point, updated